### ISSUE TITLE
Record name/properties of backup config in dlist file

### ISSUE THREAD

**Issue by robbat2** [duplicati/duplicati#1266]
As a principle, I don't commit my backups to anything without proving to myself that I can recover without the original software... so I was working on testing the new stuff, since the old instructions (https://code.google.com/p/duplicati/wiki/ManualRestoreHowTo) are very out of date.

I've got a good handle on your block-based design already, and have read Block-basedstorageformat.pdf, but there are a few pieces that I can't figure.

It looks like the dlist file does not record any of the parameters of the backup, and they could be very useful: 
  - Name of backup set
  - Destinations (possibly without passwords), esp if my issue #1265 is merged (in a disaster recovery, if you have just the dlists, they can fetch stuff back more easily).
  - Complete backup config (I'd find it very useful to have a list of paths, and the exclusions for each)
   - If you configured path 'A/' as the backup root, but the only files were in A/B/x and none in A/C/, that's important information (the empty dir in the backup).

**Comment by kenkendk** [comment]
Yes, the ManualRestoreHowTo is for the 1.3.x line, and does not apply to the 2.0 data.

In the WebUI there will be an option for storing the backup config inside the dlist files.
This is implemented, but not yet exposed to the user.

For the commandline, you can do it yourself, using the --control-files option.
Something like this:
```
CMDLINE="Duplicati.CommandLine.exe backup ssh://hostname/folder source1 source2 --option1=abc"
echo "Backup name: MyBackup" > /tmp/backup-description.txt
echo "Command: ${CMDLINE}" >> /tmp/backup-description.txt
$CMDLINE --control-files="/tmp/backup-description.txt"
```

This will add a file called "backup-description" inside the dlist zip files.
